packages/integration-sdk/CHANGELOG.md

```diff
@@ -9,6 +9,23 @@ and this project adheres to
 
 ## Unreleased
 
+### Changed
+
+- OPTIMIZATION: Buffer entities and relationships in memory and allow for fast
+  lookups. This change allows us to skip flushing to disk anytime there is a
+  call to `findEntity`, `iterateEntities` or `iterateRelationships`.
+
+- Allow `FileSystemGraphObjectStore` to specify `graphObjectBufferThreshold`,
+  which defines the maximum number of graph objects that the graph object store
+  can buffer into memory before flushing to disk. Machines with more memory
+  should consider increasing this value as the default is 500.
+
+### Removed
+
+- BREAKING: Remove `BucketMap` as it's no longer used in the
+  `FileSystemGraphObjectStore`. `BucketMap` was exposed externally, but should
+  not be used externally either.
```

Comment:
The wording here is a little confusing. Can you clarify "`BucketMap` was exposed externally, but should
  not be used externally either."?

Reply:
I had originally planned to use it in a managed graph object store, but opted not to. Nobody should be using it externally, but it's technically exposed to the external API via:

```ts
import { BucketMap } from '@jupiterone/integration-sdk-runtime';
```